src/main/java/net/dv8tion/jda/api/requests/restaction/order/OrderAction.java

```diff
@@ -177,18 +177,60 @@
      *         The new not-negative position for the currently selected entity
      *
      * @throws java.lang.IllegalStateException
-     *         If no entity has been selected yet
+     *         If no entity has been selected yet, use {@link #selectPosition(Object)}
      * @throws java.lang.IllegalArgumentException
      *         If the specified position is out-of-bounds
      *
      * @return The current OrderAction sub-implementation instance
      *
      * @see    #moveDown(int)
      * @see    #moveUp(int)
+     * @see    #moveBelow(Object)
+     * @see    #moveAbove(Object)
      */
     @Nonnull
     M moveTo(int position);
 
+    /**
+     * Moves the currently selected entity below the specified target entity.
+     *
+     * @param  other
+     *         The reference entity that should end up above the selected entity
+     *
+     * @throws IllegalStateException
+     *         If no entity has been selected yet, use {@link #selectPosition(Object)}
+     * @throws IllegalArgumentException
+     *         If the specified target entity is not managed by this instance or null
+     *
+     * @return The current OrderAction sub-implementation instance
+     *
+     * @see    #moveUp(int)
+     * @see    #moveDown(int)
+     * @see    #moveAbove(Object)
+     */
+    @Nonnull
+    M moveBelow(@Nonnull T other);
+
+    /**
+     * Moves the currently selected entity above the specified target entity.
+     *
+     * @param  other
+     *         The reference entity that should end up below the selected entity
+     *
+     * @throws IllegalStateException
+     *         If no entity has been selected yet, use {@link #selectPosition(Object)}
+     * @throws IllegalArgumentException
+     *         If the specified target entity is not managed by this instance or null
+     *
+     * @return The current OrderAction sub-implementation instance
+     *
+     * @see    #moveUp(int)
+     * @see    #moveDown(int)
+     * @see    #moveBelow(Object)
+     */
+    @Nonnull
+    M moveAbove(@Nonnull T other);
```

Comment:
The channel position field is independent of the parent_id, so it would only matter for the sorting of channels by their partial ordering through position/id.